src/server.rs

```diff
@@ -414,11 +424,11 @@ async fn index(
     payload: web::Payload,
     const_router: web::Data<Arc<ConstRouter>>,
     middleware_router: web::Data<Arc<MiddlewareRouter>>,
-    global_request_headers: web::Data<Arc<Headers>>,
-    global_response_headers: web::Data<Arc<Headers>>,
```

Comment:
please don't change this

Reply:
this was to fix ci: cargo clippy warning https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments

any other suggestions to do so?

Reply:
I meant the name.

`global_request_headers` and `global_response_headers`